api/src/test/java/uk/gov/dhsc/htbhf/claimant/entity/ClaimTest.java

```diff
@@ -86,4 +92,58 @@ void shouldReturnTheSameIdIfOneIsSet() {
         //Then
         Assertions.assertThat(id).isEqualTo(claim.getId());
     }
+
+    @Test
+    void shouldUpdateClaimStatusAndTimestamp() {
+        LocalDateTime originalTimestamp = LocalDateTime.now().minusHours(1);
```

Comment:
I think after the usage of constants for times in other tests, it would be good to create a constant in this test class called `ONE_HOUR_AGO` to use in all these tests, it'll be really clear then.